src/pheromones/hivemind-host/src/HiveMindHostSerializer.cpp

```diff
@@ -0,0 +1,22 @@
+#include "HiveMindHostSerializer.h"
+#include <pb_encode.h>
+
+HiveMindHostSerializer::HiveMindHostSerializer(IProtobufStream& stream) : m_stream(stream) {}
+
+bool HiveMindHostSerializer::serialize(const MessageDTO& message) {
```

Comment:
As @stlaperriere was saying earlier, the `serialize` name is a little confusing. I do understand this doesn't explicitly send it over the wire, but the callback, does call a send on a stream. So one way or the other (unless maybe the driver is disabled) calling `serialize()` will send data. In my mind, `serialize` would only generate a string. I know in our case with streams it doesn't work that way, but we should use a different name. Maybe `serializeAndSend()`

Reply:
Would serializeToStream/deserializeToStream be ok?